src/renderers/dom/shared/SVGDOMPropertyConfig.js

```diff
@@ -113,15 +117,19 @@ var ATTRS = [
   'xmlns:xlink',
   'xml:lang',
   'xml:space',
-  // The following attributes expect boolean values. They must be in
-  // the whitelist to allow boolean attribute assignment:
-  'autoReverse',
-  'externalResourcesRequired',
-  'preserveAlpha',
 ];
 
 var SVGDOMPropertyConfig = {
-  Properties: {},
+  Properties: {
+    autoReverse: HAS_STRING_BOOLEAN_VALUE,
+    externalResourcesRequired: HAS_STRING_BOOLEAN_VALUE,
+    preserveAlpha: HAS_STRING_BOOLEAN_VALUE,
+  },
+  DOMAttributeNames: {
+    autoReverse: 'autoReverse',
+    externalResourcesRequired: 'externalResourcesRequired',
+    preserveAlpha: 'preserveAlpha',
+  },
```

Comment:
It is curious that these must be here. This is for correct casing? I wonder if we really need to down case attributes:

https://github.com/facebook/react/blob/master/src/renderers/dom/shared/DOMProperty.js#L90

Reply:
This would be good follow-up work anyway.

Reply:
Without this, they don't get set correctly. I don't know. Maybe not?